specification/compute/resource-manager/Microsoft.Compute/RecommenderRP/models.tsp

```diff
@@ -117,3 +120,67 @@ model PlacementScore {
    */
   isQuotaAvailable?: boolean;
 }
+
+/**
+ * Lists available Capacity Blocks for a subscription.
+ */
+@added(Versions.v2025_06_05_preview)
+model CapacityBlocks is Azure.Core.Page<CapacityBlock>;
+
+/**
+ * Capacity block.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+@added(Versions.v2025_06_05_preview)
+model CapacityBlock extends Azure.ResourceManager.CommonTypes.ProxyResource {
+  /**
+   * Contains additional properties of a capacity block
+   */
+  properties?: CapacityBlockProperties;
+}
+
+/**
+ * Contains additional properties of a capacity block
+ */
+@added(Versions.v2025_06_05_preview)
+model CapacityBlockProperties {
+  /**
+   * Logical Zone of the Capacity Block
+   */
+  @visibility(Lifecycle.Read)
+  zone?: int32;
```

Comment:
I am wondering why this isn't through the top level "zones" resource property? Like it is for other zonal resources.

I understand those are usually tracked resources, and I guess that raises a question of whether this resource is a tracked resource too.

It has a location.

The only non-tracked resource thing about it seems to be that it isn't actually at resource group level?